statistics-service/src/main/java/by/ibondarau/tictactoe/statisticservice/dto/UserStatisticsDto.java

```diff
@@ -0,0 +1,16 @@
+package by.ibondarau.tictactoe.statisticservice.dto;
+
+import lombok.Data;
+import lombok.experimental.Accessors;
+
+import java.util.List;
+
+@Data
+@Accessors(fluent = true, chain = true)
```

Comment:
if you going to use it this way - you may specify it lombok.config

lombok.accessors.chain =true

lombok.accessors.fluent =true

<img width="1387" alt="image" src="https://user-images.githubusercontent.com/519785/191018018-6d696920-b192-46f9-8a18-d0bb490ddc0e.png">